src/OrchardCore.Modules/OrchardCore.Roles/Views/Admin/Edit.cshtml

```diff
@@ -52,7 +52,7 @@
                     </tr>
                 </thead>
                 <tbody>
-                    @foreach (var permission in group.Value.OrderBy(x => x.Name))
+                    @foreach (var permission in group.Value.OrderBy(x => x.Description))
```

Comment:
Why it doesn't sort the permissions alphabatically as you mentioned in the issue description?

Reply:
I don't understand the question

Reply:
> Currently, we sort permission by Name which it isn't something visible to the users. In many cases this does not show the permission in alphabetical order.

Check the last phrase

Reply:
Currently we sort by "Name" or "ID" which is something that isn't visible to the user on the UI. We show description field. So we should sort by the visible field so that the visible text is correctly sorted.

Reply:
Description could be changed, but I see sorting by name is suited here

Reply:
Yes name does not change but it's also not visible to the user. Description can change and is visible to the user, so we must sort by what is visible to the user not by hidden values. You should test this out so you can understand the problem I am solving better. Why do you think it's better to sort by hidden value in the UI rather than a visible value?

Reply:
Again description is just description, so I disagree with this change

Reply:
@hishamco 

Here is how it's sorted today, as you can see "Assign users...." permission is showing up after the 
![image](https://user-images.githubusercontent.com/24724371/190420765-036fd606-c8c0-4e0e-81e0-105c5c7aa295.png)

The reason in this case the permissions are not showing up as expected is because the permission "Assign any role" has `AssignRole` name. However, `Assign users to ....` has `ManageUsersInRole...` name.

@Skrypt the sorting is something you and I worked on a while back. Do you agree that it should be description?

Reply:
@hishamco they are grouped by feature. When this code is executed, we are sorting the permission with in a group. So the sorting is done by feature title, then by category.

Reply:
Ok, then